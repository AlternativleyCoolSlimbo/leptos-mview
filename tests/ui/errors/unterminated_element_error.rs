use leptos_mview::mview;

fn main() {
    _ = mview! {
        div {
            "something"
            input.input type="text"
        }
    };
}
