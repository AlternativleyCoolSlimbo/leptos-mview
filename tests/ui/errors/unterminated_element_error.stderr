error: unterminated element
 --> tests/ui/errors/unterminated_element_error.rs:7:13
  |
7 |             input.input type="text"
  |             ^^^^^^^^^^^^^^^^^^^^^^^
  |
  = help: add a `;` to terminate the element with no children
